### PR TITLE
Added liquibase note to changelog.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,4 +11,12 @@
 }
 ```
 * Changed the JES runtime attributes `defaultDisks` and `defaultZones` to be simply `disks` and `zones` respectively.
-
+* Liquibase scripts now run automatically. Cromwell won't start if it detects evidence of manual migrations in a
+persistent database. Instead, before cromwell will start cleanly, the database should backed up, and then this SQL
+should be manually executed:
```
Comment (cjllanwarne) [inline CHANGELOG.md]: Could you mention who needs to do this? E.g. if I have a local database, do I need to run this? Or only if I initialised it with liquibase?

**Comment by kshakir** [comment]
Text updated

**Comment by kcibul** [comment]
lgtm - thanks Khalid

**Comment by cjllanwarne** [comment]
:+1:

**Comment by geoffjentry** [comment]
:+1:

**Pull request merged**